frontend/src/components/data-table/columns.tsx

```diff
@@ -110,6 +114,9 @@ export function generateColumns<T>({
   textJustifyColumns?: Record<string, "left" | "center" | "right">;
   wrappedColumns?: string[];
   showDataTypes?: boolean;
+  calculateTopKRows?: CalculateTopKRows;
+  setFilters?: SetFilters;
+  filters?: ColumnFiltersState;
```

Comment:
passing these down will cause re-render to the columns. instead, these can be grabbed off the `column`  object in `header: ({ column }) => {`